compile_testing.py

```diff
@@ -0,0 +1,67 @@
+import os, shutil
+
+excluded_folders = [
+    "multimodal_clip"
+]
+
+try:
+    os.mkdir("testing-folder")
+except:
+    pass
+
+root = "./examples"
+
+requirements = []
+commands = []
+
+# For each folder in the examples folder
+for folders in os.listdir(root):
+    folder = os.path.join(root, folders)
+    if os.path.isdir(folder) and folders not in excluded_folders:
+        for file in os.listdir(folder):
+            # If the file is a requirements.txt file, add it to the list of requirements
+            if file.endswith("requirements.txt"):
+                with open(os.path.join(folder, file), "r") as f:
+                    requirements.extend(f.read().splitlines())
+            # If the file is a python file, copy it to the testing folder
+            if file.endswith(".py"):
```

Comment:
Is this right? Now that there is a test.py for each example you probably don't need to copy the any other .py file anywhere right? You can simply check if file is test.py and run it right there no?

Reply:
Yes, I just left the python file as is, and only changed the github actions